101-batch-with-keyvault-pfx-password/azuredeploy.parameters.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "batchservice": {
+      "value": "GET-PREREQ-batchservice"
+    },
+    "certdata": {
+     "value": "GEN-SELFSIGNED-CERT-PFXDATA"
+    },
+    "certthumbprint": {
+      "value": "5384a8be6df0f82717d08e18028636aac668b6a7"
```

Comment:
use the GEN value for the thumbprint here

Reply:
Done